README.md

```diff
@@ -8,7 +8,7 @@ Example docker setup to run a nats server, a nats-go listener, and a nats-go req
 ```
 apt-get install golang # https://wiki.ubuntu.com/Go
 export GOPATH=`pwd`
-go get github.com/nats-io/go-nats
+go get shadiakiki1986/example-nats-go-docker
```

Comment:
Thanks for the PR, but this line yields the following for me
```bash
> go get shadiakiki1986/example-nats-go-docker 
package shadiakiki1986/example-nats-go-docker: unrecognized import path "shadiakiki1986/example-nats-go-docker" (import path does not begin with hostname)
```
What does it output for you?